README.md

```diff
@@ -88,15 +88,15 @@ interactive WebVR experiences.
 
 :globe_with_meridians: **Cross-Platform**: Build VR applications for Vive,
 Rift, Daydream, GearVR, and Cardboard. Don't have a headset or controllers? No
-problem! A-Frame still works on standard desktop and smartphones.
```

Comment:
when talking about software. we usually say this software is made for desktop and mobile. Singular is usually used.